test/fw/ptl/utils/plugins/ptl_test_data.py

```diff
@@ -150,23 +150,26 @@ def __save_home(self, test, status, err=None):
             f.close()
             return
 
-        server = getattr(_test, 'server', None)
+        servers = getattr(_test, 'servers', None)
         moms = getattr(_test, 'moms', None)
         comms = getattr(_test, 'comms', None)
-        client = getattr(_test.server, 'client', None)
+        client = getattr(_test.servers, 'client', None)
         add_hosts = []
+        if len(servers) > 1:
+            for param in servers.values():
+                add_hosts.append(param.shortname)
         if moms is not None:
             for param in moms.values():
                 add_hosts.append(param.shortname)
         if comms is not None:
-            for param_dict in comms.values():
+            for param in comms.values():
                 add_hosts.append(param.shortname)
         if client is not None:
             add_hosts.append(client.split('.')[0])
         add_hosts = list(dict.fromkeys(add_hosts))
         add_hosts = [host for host in add_hosts if host != current_host]
-        if server is not None:
-            server_host = server.hostname
+        if servers is not None:
+            server_host = (servers.values()[0]).shortname
```

Comment:
any particular reason why you are using 'shortname' ? I think 'hostname' will be safer to use, no? Same comment for other instances

Reply:
Yes, if I pass the hostname then snapshot logs will be saved with FQDN name on some machines which is not a good practice to save logs with full hostname.